scripts/firedrake-install

```diff
@@ -16,6 +16,15 @@ from glob import iglob
 from itertools import chain
 import re
 import importlib
+import warnings
+
+
+warnings.warn(
+    "firedrake-install/firedrake-update is no longer the recommended way to "
+    "install/update Firedrake. Please use pip instead. For instructions on how "
+    "to do this please visit https://www.firedrakeproject.org/download.html.",
+    FutureWarning
+)
```

Comment:
Maybe put a 10 second pause in after this warning so that users have some chance of seeing it.

Reply:
Excellent point!